pkg/lib/packagekit.js

```diff
@@ -612,6 +613,17 @@ export async function get_updates(details, progress_cb) {
                 if (info < Enum.INFO_LOW || info > Enum.INFO_SECURITY)
                     info = Enum.INFO_NORMAL;
 
+                if (info == Enum.INFO_LOW)
```

Comment:
Hmm , strictly speaking, this kind of conversion should happen in PackageManager. If you do it here, it's a API break and you had to adapt pkg/packagekit for that. Are we worried about not breaking users of pkg/lib/packagekit?